openmc/settings.py

```diff
@@ -66,6 +67,16 @@ class Settings:
         history-based parallelism.
 
         .. versionadded:: 0.12
+    frequency_mesh : openmc.RegularMesh
+        Mesh to be used to set the flux or precursor frequency
+    frequency_group_structure : openmc.mgxs.EnergyGroups
+        Energy group structure used for setting the flux frequency
+    frequency_num_delayed_groups : int
+        Number of delayed groups for precursor frequency
+    flux_frequency : np.ndarray
+        Array of flux frequencies of size frequency_mesh x frequency_energy_groups
+    precursor_frequency : np.ndarray
+        Array of precursor frequencies of size frequency_mesh x frequency_delayed_groups
```

Comment:
I would prefer to see these settings be added in a pull request where they are actually used rather than this one (where they don't appear to have any purpose).

Reply:
I agree with you! They would fit better in a separate PR so I took them out. I did however leave the frequency_mesh since the presence of the frequency_mesh is necessary to turn frequency_method_on to True.